sw/device/lib/testing/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Chip-Level Test Libraries"
+---
+
+This subtree provides libary code for writing chip-level tests.
+Test library code should be considered any (reusable) code that could aid in the writing of chip-level tests.
+
+## Test Types
+- **Chip-level Tests** - A collection of software level tests that run on OpenTitan hardware, whose main purpose is pre-silicon verification and post-silicon bringup.
+These tests consist of: smoke, IP integration, and system-level tests.
+These tests are all executed on-device using the [on-device test framework]({{< relref "sw/device/lib/testing/test_framework/index.md" >}}).
```

Comment:
Are we changing the terminology to ODTF from OTTF?

Reply:
"on-device test framework" is what is currently in place. OTTF will (hopefully) replace it.

Reply:
I see, ok.